packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx

```diff
@@ -0,0 +1,121 @@
+import { useMemo, useState } from 'react'
+
+import { FlexBox } from '@/components/FlexBox/FlexBox'
+import { Information } from '@/components/Information'
+import { JoyTokenIcon } from '@/components/JoyTokenIcon'
+import { NumberFormat } from '@/components/NumberFormat'
+import { Text } from '@/components/Text'
+import { TextButton } from '@/components/_buttons/Button'
+import { FormField } from '@/components/_inputs/FormField'
+import { TokenInput } from '@/components/_inputs/TokenInput'
+import { DetailsContent } from '@/components/_nft/NftTile'
+import { DialogModal } from '@/components/_overlays/DialogModal'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+export type SellTokenModalProps = {
+  tokenId: string
+  onClose: () => void
+}
+
+const getTokenDetails = (_: string) => ({
+  title: 'JBC',
+  pricePerUnit: 1000,
+  tokensOnSale: 67773,
+  userBalance: 100000,
+})
+
+export const SellTokenModal = ({ tokenId, onClose }: SellTokenModalProps) => {
+  const [tokens, setTokens] = useState<number | null>(null)
+  const smMatch = useMediaMatch('sm')
+  const { pricePerUnit, tokensOnSale, userBalance, title } = getTokenDetails(tokenId)
+
+  const details = useMemo(
+    () => [
+      {
+        title: 'You will get',
+        content: (
+          <NumberFormat
+            value={tokens || 0}
+            as="p"
+            variant="t200"
+            withDenomination="before"
+            withToken
+            customTicker="$JBC"
```

Comment:
you're selling $JBC here, so you should receive `${atlasConfig.joystream.tokenTicker}`?

Reply:
same thing with align as in token buy modal